packages/EasyUtils/src/Interfaces/MathInterface.php

```diff
@@ -35,8 +35,13 @@ public function abs(string $value, ?int $precision = null, ?int $mode = null): s
 
     public function add(string $augend, string $addend, ?int $precision = null, ?int $mode = null): string;
 
+    /**
+     * @deprecated must be deleted on next major update, use compareThat() method instead
```

Comment:
I don't think we should deprecate this method at all. The new methods are just "sugar" ones, why should we force anyone to use them?

Reply:
Agree, removed deprecation